Source/WebCore/platform/graphics/cairo/ImageBufferUtilitiesCairo.cpp

```diff
@@ -65,7 +65,7 @@ static bool encodeImage(cairo_surface_t* image, const String& mimeType, Vector<u
     return cairo_surface_write_to_png_stream(image, writeFunction, output) == CAIRO_STATUS_SUCCESS;
 }
 
-Vector<uint8_t> data(cairo_surface_t* image, const String& mimeType, std::optional<double>)
+Vector<uint8_t> encodeData(cairo_surface_t* image, const String& mimeType, std::optional<double>)
```

Comment:
This definition is inside a `#if !PLATFORM(GTK)` but the method is also needed in the GTK port, so this breaks the build as well.